batch/batch.go

```diff
@@ -119,15 +119,18 @@ func (b *Builder) Build(now time.Time) (*Batch, error) {
 
 var (
 	CorruptedBatchError = errors.New("batch crc does not match bytes")
+	UnsupportedProtocolError = errors.New(" protocol version with magic value 2 is only supported")
```

Comment:
" protocol version with magic value 2 is only supported" -> "batch magic value is not 2"